docs/releases/5.1.1.txt

```diff
@@ -18,3 +18,7 @@ Bugfixes
   children). A new :class:`~django.contrib.auth.forms.AdminUserCreationForm`
   including this field was added, isolating the feature to the admin where it
   was intended (:ticket:`35678`).
+
+* Restored, following a regression in Django 5.1, the ability to override the
+  timezone and role setting behavior used within the ``init_connection_state``
+  method of PostgreSQL backends (:ticket:`35688`).
```

Comment:
Should backend be singular? Perhaps
```suggestion
  method in the PostgreSQL backend (:ticket:`35688`).
```

Reply:
Ah I thought maybe I should worry that postgis inherits this but it seems `PostgreSQL backends` is never used so updated it to be singular :+1: